model/Core/Classes/SpdxDocument.md

```diff
@@ -23,9 +23,15 @@ Commonly used when representing a unit of transfer of SPDX Elements.
   - type: xsd:string
   - minCount: 1
   - maxCount: 1
-
-## External properties restrictions
-
-- /Core/Element/name
+- element
```

Comment:
isn't a `Element` with the list of elements a Bundle or Collection? Why not inherit from one of those?

Reply:
In general because SpdxDocument has nothing to do with the logical model, or as Sean put it yesterday, with the "SPDX graph". When you define a Bundle element you are saying "the graph is saying something meaningful and worth remembering about this group of elements", whereas SpdxDocument is saying "if you have a payload or find it based on locationHint, you can verify the elements in it".  The fact that some elements are in the same payload is not worth memorializing in the graph, particularly since the same elements can appear in different payloads in different combinations."   ElementCollection is a persistent grouping, the `SpdxDocument Element` is purely transactional, as opposed to the `SpdxDocument` (payload) itself, which is a concrete persistent object itself that has meaning and is used at the physical layer, outside the graph, regardless of whether an `SpdxDocument Element` has ever been created to reference it.

In particular, because even ElementCollection has rootElement, which doesn't have anything to do with serialized data payloads.  The payload itself has to contain everything needed to understand and use it; anything in the `SpdxDocument Element` beyond what is needed to find and verify serialized data is just another dependency / coupling between logical and physical layers that should not exist.